thoth/storages/ceph.py

```diff
@@ -97,6 +99,15 @@ def retrieve_blob(self, object_key: str) -> bytes:
                 raise NotFoundError("Failed to retrieve object, object {!r} does not exist".format(object_key)) from exc
             raise
 
+    def retrieve_document_last_modification_date(self, object_key: str) -> datetime:
+        """Retrieve the last modification date of a document from S3."""
+        try:
+            return self._s3.Object(self.bucket, f"{self.prefix}{object_key}").get()["LastModified"]
```

Comment:
Since the previous comment would make this a single line, maybe we could just drop the method and have the caller use `['LastModified'] on the object` ?